application-src/amsamba.pl

```diff
@@ -510,9 +510,9 @@ sub command_estimate {
 	    push @ARGV, "-D", $self->{subdir},
 	}
 	if ($level == 0) {
-	    push @ARGV, "-c", "archive 0;recurse;du";
+	    push @ARGV, "-c", "archive 1;recurse;du /" . $self->{subdir};
```

Comment:
Why this change of "archive 0" to "archive 1"?

Reply:
I'm not sure why either...

Reply:
I found reasoning in #76.

Reply:
It doesn't seem to say why the "archive 0" was changed to archive 1, does it?

I put a new pull-req in to use "du *" because that seems to work as intended, recursively.